hail/python/hail/experimental/vcf_combiner/vcf_combiner.py

```diff
@@ -512,13 +512,27 @@ def run_combiner(sample_paths: List[str],
                  batch_size: int = CombinerConfig.default_batch_size,
                  target_records: int = CombinerConfig.default_target_records,
                  import_interval_size: Optional[int] = None,
-                 is_genomes: bool = False,
+                 use_genome_default_intervals: bool = False,
+                 use_exome_default_intervals: bool = False,
                  overwrite: bool = False,
                  reference_genome: str = 'default',
                  contig_recoding: Optional[Dict[str, str]] = None,
                  key_by_locus_and_alleles: bool = False):
     """Run the Hail VCF combiner, performing a hierarchical merge to create a combined sparse matrix table.
 
+    **Partitioning**
```

Comment:
Consider adding something about how the input partition and output partitions are unrelated unless number of gvcfs is less than the branch_factor.

Reply:
As a followup, but may be unnecessary, discuss how smaller and larger numbers of import partitions affect performance. Notably increased parallelism at the cost of re-parsing tabix indices every time and weird object storage performance properties where GCS 'warms up' and can seem to get a little faster overall on long reads.

These come from my observations that as parallelism decreased, core hours decreased, in many cases significantly.

Reply:
This is hard, since I think we don't understand the relationship super well. I think we should communicate that people should generally use the default_$YOUR_DATA_TYPE_intervals